indra/llcharacter/llik.cpp

```diff
@@ -3314,7 +3335,8 @@ void LLIKConstraintFactory::initSingleton()
     {
         std::ifstream file(constraint_file);
 
-        if (!LLSDSerialize::deserialize(constraint_data, file, LLSDSerialize::SIZE_UNLIMITED))
+        constexpr llssize MAX_EXPECTED_LINE_LENGTH = 256;
+        if (!LLSDSerialize::deserialize(constraint_data, file, MAX_EXPECTED_LINE_LENGTH))
```

Comment:
Yeah so... `LLSDSerialize::SIZE_UNLIMITED` doesn't actually work.  It wasn't clear to me if it was failing because of how we were actually using it when we tried to parse each line of the file, or if it should never EVER be used, but the result was that the constraints config file would fail to load anything.  Consequentially the IK was happening on unconstrained chains and this, combined with some other bugs, resulted in the horrifying spinning IK arms.  I'm introducing this `MAX_EXPECTED_LINE_LENGTH` value as a quick way to solve the problem without falling down a rabbit hole in the **LLSD** parsing code.